basic_samples/Dataviews/Python3/LICENSE

```diff
@@ -0,0 +1,1242 @@
+Copyright 2020
+
+[product-readiness.DataViews_Python : merge]
```

Comment:
I assume this is here because it was generated from the merge branch in the BlackDuck project, but it seems like this should be just `DataViews_Python`, and likewise for the other license files. I don't know whether we should modify the generated files, but the name here won't make sense to anyone external.